packages/react-vapor/src/components/select/hoc/SelectWithInfiniteScroll.tsx

```diff
@@ -8,6 +8,7 @@ import {ISelectOwnProps} from '../SelectConnected';
 export interface SelectWithInfiniteScrollProps {
     totalEntries: number;
     next: () => void;
+    scrollThreshold?: React.ReactText;

```

Comment:
This is pretty technical and users of the component will have to go [`react-infinit-scroll-component`'s documentation](https://www.npmjs.com/package/react-infinite-scroll-component#props) to understand what it does. I suggest we remove this prop and change the value to something more forgiving instead. Have you tried `0.9`? or not specifying it entirely to use the default value of `0.8`?